### PR TITLE
modify jobench and add unittest

### PR DESCRIPTION
**Pull request by arzuschen**
Remove unnecessary code in jobench unittest and add jobench stats import for program.cs.
Add codegen unittest to cover left NLJoin, antisemi NLJoin and null groupby_ for aggregation.

### CODE REVIEW

File: test/UnitTest.cs [hunk 1]
```diff
@@ -297,10 +302,6 @@ public void TestJobench()
             string write_dir_fn = $"../../../../test/regress/output/jobench";
             string expect_dir_fn = $"../../../../test/regress/expect/jobench";
 
-            // make sure all queries can generate phase one opt plan
-            QueryOption option = new QueryOption();
-            option.optimize_.TurnOnAllOptimizations();
```
Comment (zhouqingqing) [inline test/UnitTest.cs:302]: Please keep this line as it turns on all optimizations for jobench run.

Comment (arzuschen) [reply test/UnitTest.cs:302]: I think this option is not used, the option has optimize turned on in psql/Program.cs line 31

Comment (zhouqingqing) [reply test/UnitTest.cs:302]: You are right. In a separate patch, can you please remove psql project and move its content to unittest? We used to have psql designed as a SQL client but we don't think we will need it anymore.

Comment (arzuschen) [reply test/UnitTest.cs:302]: got it

**Pull request closed without merging**